src/scss/components/_buttons.scss

```diff
@@ -5,19 +5,19 @@
   line-height: 40px;
   padding: 0 14px;
   box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
-  background: $primary;
```

Comment:
Will do!

Reply:
Actually, that would kind of mess up the design because I had changed $primary to black and in this case, I wanted to change the colour to light-blue

Reply:
so basically the point I was trying to make was lets have variables for all the colours we need:
so $primary would be black, $secondary could be light-blue etc. This is just a good practise to have because in the future if you need to change the colors, instead of having to change all the fonts  and colors everywhere all you have to do it change the variables value. Do you see what i mean?

Reply:
Yeah gotcha!